mlir/include/mlir/Conversion/Passes.td

```diff
@@ -578,7 +578,10 @@ def ConvertGPUToSPIRV : Pass<"convert-gpu-to-spirv", "ModuleOp"> {
     to control the set and binding if wanted.
   }];
   let constructor = "mlir::createConvertGPUToSPIRVPass()";
-  let dependentDialects = ["spirv::SPIRVDialect"];
+  let dependentDialects = [
+    "spirv::SPIRVDialect",
+    "func::FuncDialect",
```

Comment:
nit: IIRC we try to keep these sorted alphabetically
```suggestion
    "func::FuncDialect",
    "spirv::SPIRVDialect",
```

Reply:
Done.